ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,72 @@
+Issue Title
+===========
+
+Your issue title is above. Please include the path to the
+topic with the issue. For example, if your issue is that
+our [list of samples](https://stage.docs.microsoft.com/en-us/dotnet/articles/samples-and-tutorials/index)
+is missing a sample, you'd put [samples-and-tutorials/index.md]
+in the issue title.
+
+If your issue is that a topic is missing, please preface
+the issue title with [NewTopic]. That helps us find requests
+for new topics.
+
+General
+=======
+
+We use issues to drive the discussion for changes to
+existing topics, and the creation of new topics.
+
+This way, we have those discussions before significant
+effort is spent writing a new topic. Please see our
+Contributing.md guidelines for details.
+
+Issues with Existing Topics
+===========================
+
+Please describe the problem with the current topic. Be as
+specific as you can. 
+
+- Is content that you wanted missing?
+- Is content incorrect?
+- Are links broken?
+- Is some content not relevant to the topic?
+
```

Comment:
Maybe you could add: Are we missing a scenario for a particular topic? Do we need to add samples or images to a topic?